test/Rings/mpolyquo-localizations.jl

```diff
@@ -510,3 +510,17 @@ end
   @test Oscar._get_generators_string_one_line(ideal(R, x)) == "with 100 generators"
   @test Oscar._get_generators_string_one_line(ideal(R, [sum(x)])) == "with 1 generator"
 end
+
+@testset "monomial_basis MPolyQuoLocRing" begin
+  R, (x,y) = QQ["x", "y"]
+  L, _ = localization(R, complement_of_point_ideal(R, [0,0]))
+  Q1, _ = quo(L, ideal(L, L(x+1)))
+  @test isempty(monomial_basis(Q1))
+  Q2, _ = quo(L, ideal(L, L(x^2)))  
+  @test_throws InfiniteDimensionError() monomial_basis(Q2)
+  Q3, _ = quo(L, ideal(L, L.([x^2, y^3])))  
+  @test monomial_basis(Q3) == L.([x*y^2, y^2, x*y, y, x, 1])
+  L1, _ = localization(R, complement_of_point_ideal(R, [1,2]))
+  @test monomial_basis(L1, ideal(L1, [(x-1)^2, (y-2)^2])) == L1.([x*y, y, x, 1])  
+  @test isempty(monomial_basis(L1, ideal(L1, L1.([x, y]))))
+end
```

Comment:
Please add one or two non-homogeneous cases to the tests, so that the difference between localization and non-localized ring will be detected and unexpected behaviour (by use of wrong ordering, as remarked by @ederc above) would pop up as an error.

Reply:
just added the additional tests